update.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -eux
+
+./build.py
+
+if [ -z "$(git diff --exit-code)" ]; then
+    echo "No changes to the output on this push; exiting."
+    exit 0
+fi
+
+find .
+
+export VERSION=$(cat version)
+
+export GIT_AUTHOR_NAME="Github update bot"
+export GIT_AUTHOR_EMAIL="git@github.com"
+export GIT_COMMITTER_NAME=$GIT_AUTHOR_NAME
+export GIT_COMMITTER_EMAIL=$GIT_AUTHOR_EMAIL
+
+git add --all .
```

Comment:
I would prefer to only add the files we expect to be updated so I don't accidentally push a bunch of files.

Reply:
Restricted to the version file and the _data.py.